src/de.rs

```diff
@@ -434,6 +437,7 @@ struct DeserializerFromEvents<'de, 'document> {
     path: Path<'document>,
     remaining_depth: u8,
     current_enum: Option<CurrentEnum<'document>>,
+    serde_content_newtype: bool,
```

Comment:
_(You can push back on this comment)_

I'm not sure about this name. TBH honest, I don't understand what this field does. You don't have to add a comment because it would be too noisy, but I like to name booleans as a yes-or-no question. This is the same thing you did down there with "has visited?" Could you do this here as well?

What is this? `is_serde_content_newtype`? `contains_newtype`? something else?

Or is this boolean not a switch, but some other semantic value?

Reply:
It just a `copy-paste` from `ron`'s hacky. From my perspective, `serde_content_newtype` or `is_serde_content_newtype` ensures the hacky is just applied to the first `TaggedValue`, if `V::Value` is serde internal buffer.